sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -47,364 +48,444 @@ const alwaysBeDeletedSubscription = "alwaysbedeletedsubscription";
+      );
+    });
+
+    it(`Creating an existent ${testCase.entityType} entity throws an error`, async () => {
+      let response;
+
+      try {
+        await createEntity(
+          testCase.entityType,
+          testCase.alwaysBeExistingEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        response = err;
+      }
+
+      should.equal(response.statusCode, 409, "Error must not be undefined");
+      should.equal(
+        response.message.startsWith("The messaging entity") ||
+          response.message.startsWith("Entity") ||
+          response.message.startsWith("SubCode") ||
+          response.message.startsWith("No service"),
+        true,
+        `Expected error message to be a textual content but got "${response.message}"`
+      );
     });
-  }
-);
+
+    it(`Lists available ${testCase.entityType} entities successfully`, async () => {
+      let response;
+
+      response = await listEntities(
+        testCase.entityType,
+        testCase.parentTopicName,
+        testCase.parentSubscriptionName
+      );
+
+      should.equal(Array.isArray(response), true, "Result must be any array for list requests");
+    });
+
+    it(`Updates an existent ${testCase.entityType} entity successfully`, async () => {
+      let response: any;
+
+      response = await updateEntity(
+        testCase.entityType,
+        testCase.alwaysBeExistingEntity,
+        testCase.parentTopicName,
+        testCase.parentSubscriptionName
+      );
+      should.equal(
+        response[testCase.entityType.toLowerCase() + "Name"],
+        testCase.alwaysBeExistingEntity,
+        "Entity name mismatch"
+      );
+    });
+
+    it(`Update on non-existent ${testCase.entityType} entity throws an error`, async () => {
+      let response;
+
+      try {
+        await deleteEntity(
+          testCase.entityType,
+          testCase.alwaysBeDeletedEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        log.httpAtomXml("Ignoring clean up step");
+      }
+      try {
+        await updateEntity(
+          testCase.entityType,
+          testCase.alwaysBeDeletedEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        response = err;
+      }
+
+      should.equal(response.statusCode, 404, "Error must not be undefined");
+      should.equal(
+        response.message.startsWith("The messaging entity") ||
+          response.message.startsWith("Entity") ||
+          response.message.startsWith("SubCode") ||
+          response.message.startsWith("No service"),
+        true,
+        `Expected error message to be a textual content but got "${response.message}"`
+      );
+    });
+
+    it(`Gets an existent ${testCase.entityType} entity successfully`, async () => {
+      let response;
+
+      response = await getEntity(
```

Comment:
Same here, how do we know the entity already exists?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/6112#issuecomment-555716170